CHANGELOG.md

```diff
@@ -28,6 +28,25 @@ The changelog is generated during the release process using [ChangelogLinker](ht
 
 <!-- changelog-linker -->
 
+## [7.0.0-beta1] - 2018-09-18
+
+### [shopsys/framework]
+
+#### Changed
+
+- [#472] Changed controllers and factories "private" methods and attributes to "protected" to allow extensibility
+
+#### Fixed
+
+- [#480] admin: fix product form for product variants
```

Comment:
Missing: [#473] Fixed rendering forms in extended admin controller to be as same as in framework